### ISSUE TITLE
The machine is in the 'gurumeditation' state

### ISSUE THREAD

**Issue by technhoj** [splunk/attack_range#1015]
Hi,

I always encounter below error when trying to build. Any help is appreciated.

> The guest machine entered an invalid state while waiting for it
> to boot. Valid states are 'starting, running, paused'. The machine is in the
> 'gurumeditation' state. Please verify everything is configured
> properly and try again.
> 
> If the provider you're using has a GUI that comes with it,
> it is often helpful to open that and watch the machine, since the
> GUI often has more helpful error messages than Vagrant can retrieve.
> For example, if you're using VirtualBox, run `vagrant up` while the
> VirtualBox GUI is open.
> 
> The primary issue for this error is that the provider you're using
> is not properly configured. This is very rarely a Vagrant issue.[0m
> ERROR - attack_range - vagrant failed to build

**Comment by P4T12ICK** [comment]
Hello, can you try to provide more information? What host OS do you use? Can you try to get more information form the GUI?

**Comment by technhoj** [comment]
Hi
I was using Ubuntu Server 24.04.1 LTS VM inside Proxmox.
Virtualbox 7.0
Vagrant 2.4.3-1

**Comment by P4T12ICK** [comment]
I would need to have more information otherwise I can't help you. I can't reproduce it.

**Comment by technhoj** [comment]
![attack_range](https://github.com/user-attachments/assets/987d3c74-3372-4a3d-9e35-9954ab3e2a67)

I have attached the screenshot of the error. Hope that will help.

**Comment by P4T12ICK** [comment]
I don't think this has anything to do with Attack Range. I think it is related to your Virtualbox installation in combination with your OS.
You need to open Virtualbox and get more information from there.

**Comment by technhoj** [comment]
managed to resolved the issue after I increased the VM specs.
However, i faced another issue. Any idea how to resolve this?
![image](https://github.com/user-attachments/assets/592b4553-444f-4cfe-825a-b26c9456324d)

**Comment by technhoj** [comment]
can advise which file should i modify to add a retry if it encountered this error?

**Comment by mds-admin** [comment]
Could you please let me know how you fixed this error? I’ve updated the VM specs, but I’m still getting the same error.

Error: The guest machine entered an invalid state while waiting for it to boot. Valid states are 'starting', 'running', 'paused'. The machine is in the 'gurumeditation' state. Please verify everything is configured properly and try again.'

Any insights would be appreciated.

**Comment by technhoj** [comment]
I modified the vagrantfiles to double the specs.
